packages/expo-constants/src/Constants.types.ts

```diff
@@ -272,17 +272,22 @@ export interface NativeConstants {
    */
   nativeBuildVersion: string | null;
   /**
-   * Classic manifest for Expo apps using classic updates.
-   * Returns `null` in bare workflow and when `manifest2` is non-null.
+   * Classic manifest for Expo apps using classic updates. Returns `null` in bare workflow and when
+   * `manifest2` is non-null.
+   * > Prefer using `Constants.expoConfig` instead, which behaves more consistently across classic
```

Comment:
Do we want to mention 'builds' here as well? Most common confusion is: using Constants.manifest which works fine for a build, then pushing an update with EAS Update, and it turning `null`.

Reply:
I think we can add that detail for now. It might change as the embedded update becomes more similar to downloaded updates.